dashboard/app/models/user.rb

```diff
@@ -395,6 +396,10 @@ def hide_email_and_full_address_for_students
     end
   end
 
+  def hide_school_info_for_students
+    self.school_info = nil if student?
+  end
```

Comment:
Is it worth mashing this into `hide_email_and_full_address_for_students` (with an appropriate change to the method name)? It seems to me it would be, as I don't think we'd ever want to do that without this.